src/tap/src/tapcell.cpp

```diff
@@ -2,7 +2,7 @@
 //
 // BSD 3-Clause License
 //
-// Copyright (c) 2019, The Regents of the University of California
+// Copyright (c) 2019, University of California, San Diego.
```

Comment:
Please don't change this

Reply:
Sorry, I did not notice, they apparently get changed automatically whenever I pull, reverted now.